scanner/tlstarget.go

```diff
@@ -433,8 +438,10 @@ func (h *CertHostTLSTarget) Dump(hostFh, certFh, chrFh, scsvFh, httpFh *os.File,
 				}
 
 				// Write to certificate-host relation CSV file
-				// [cert_hash, host, port, depth]
-				if ok := chrCsv.Write([]string{sha256Hex, ip, port, h.domain, strconv.Itoa(i)}); ok != nil {
+				// [cert_hash, host, port, depth, pub_key_hash]
+				sha2PubKey := hex.EncodeToString(getSHA256(cert.RawSubjectPublicKeyInfo))
+
+				if ok := chrCsv.Write([]string{sha256Hex, ip, port, h.domain, strconv.Itoa(i), sha2PubKey}); ok != nil {
```

Comment:
I'd personally change the variable name to something like `sha256SPKI`. Other than that, this PR LGTM :+1:

Reply:
ok changed it and going to merge now